server/command_hooks.go

```diff
@@ -82,6 +82,45 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 			ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL,
 			Text:         "Disabled demo plugin hooks.",
 		}, nil
+	} else if strings.HasSuffix(args.Command, "ephemeral") {
+
+		URL := fmt.Sprintf("%s", *p.API.GetConfig().ServiceSettings.SiteURL)
+
+		post := &model.Post{
+			ChannelId: args.ChannelId,
+			UserId:    args.UserId,
+			Message:   "test ephemeral actions",
+			Props: model.StringInterface{
+				"attachments": []*model.SlackAttachment{
+					{
+						Actions: []*model.PostAction{
+							{
+								Id: model.NewId(),
```

Comment:
Is this needed? I don't think so.

Reply:
Not needed.  Removed Id from each `PostAction`